lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -120,4 +120,14 @@ void rtc_clear_wakeup_flag(void)
 	RTC_ISR &= ~RTC_ISR_WUTF;
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Clears the Alarm A flag
+
+*/
+void rtc_clear_alarm_flag(void)
+{
+	/* clear flag: not write protected */
```

Comment:
that sort of comment is probably more useful above perhaps? where it makes it into the doxygen?